src/main/java/uk/gov/hmcts/reform/sscs/utility/dwpResponseUtil.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.sscs.utility;
```

Comment:
would this be needed in any other repo? if not, why not just put it in tribunals API